src/geometry/euclidean/ceva_theorem.lean

```diff
@@ -0,0 +1,325 @@
+/-
+Copyright (c) 2021 Mantas Bakšys. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Mantas Bakšys
+-/
+import tactic.linarith
+import tactic.fin_cases
+import tactic.norm_num
+import geometry.euclidean
+import linear_algebra.affine_space.basis
+import tactic.basic
+import analysis.normed_space.add_torsor_bases
+import data.equiv.basic
+import group_theory.perm.list
```

Comment:
```suggestion
import geometry.euclidean
import linear_algebra.affine_space.basis
import analysis.normed_space.add_torsor_bases
import group_theory.perm.list
```
Since imports are transitive, it's not necessary to import basic stuff here. It's a perfectly safe assumption that standard tactics are imported already in e.g. `analysis.normed_space.add_torsor_bases` :)